clients/unshaded/src/main/java/tachyon/client/table/AbstractTachyonRawTables.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client.table;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+
+import tachyon.TachyonURI;
+import tachyon.annotation.PublicApi;
+import tachyon.client.RawTableMasterClient;
+import tachyon.exception.TachyonException;
+import tachyon.thrift.RawTableInfo;
+
+/**
+ * Tachyon Raw Table client. This class should be used to interface with the Tachyon Raw Table
+ * master.
+ */
+@PublicApi
+public class AbstractTachyonRawTables implements TachyonRawTablesCore {
+  protected RawTablesContext mContext;
+
+  protected AbstractTachyonRawTables() {
```

Comment:
I'm not touching the names for now, just keeping consistent with the previous naming.